packages/tocco-ui/src/ButtonGroup/StyledButtonGroup.js

```diff
@@ -0,0 +1,20 @@
+import styled from 'styled-components'
+import {theme} from 'styled-system'
+
+const StyledButtonGroup = styled.div`
+  && {
+    display: flex;
+    flex-flow: row wrap;
+    width: fit-content;
+    margin-bottom: -${props => props.melt ? 0 : theme('space.4')};
+
+    a,
+    button {
+      margin-bottom: ${props => props.melt ? 0 : theme('space.4')};
+    }
+
+    border-radius: ${props => props.melt === true ? theme('radii.3') : 0};
```

Comment:
use consistent checks for `props.melt` in this file (there is no `=== true` on lines 9 and 13). IMO you don't have to write `=== true`, because the prop type declaration makes sure that it is a boolean.

Reply:
see 6f3a3d106a400cb2e0c7d94125948c6ba661cc5d